### PR TITLE
docs(batch-operations): improve documentation for "Set a Removal Time"

### PR DESCRIPTION
**Pull request by valeriaportolesicamunda**

### CODE REVIEW

**Comment by yanavasileva** [comment]
Related to https://github.com/camunda/camunda-bpm-platform/issues/2792

**Comment by yanavasileva** [comment]
@valeriaportolesicamunda please note the changes should be reviewed by Automation Platform team (owner of the repo).

**Comment by garima-camunda** [comment]
Thanks @yanavasileva . I have removed my name from the Reviewer list. Feel free to continue.

@valeriaportolesicamunda, we can discuss the process separately.

**Comment by yanavasileva** [comment]
@garima-camunda, you can also add your input as reviewer.
@valeriaportolesicamunda please let me know the PR is ready and I can pick it up for a review on our side.

**Comment by valeriaportolesicamunda** [comment]
@yanavasileva I confirm PR is ready

File: content/user-guide/process-engine/batch-operations.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ The following modes can be chosen:
 
 Historic process and decision instances can be part of a hierarchy. To set the same removal time for all instances within
 a hierarchy, the method `.hierarchical()` needs to be called.
+Setting removal time to running process instances would delete data from historic tables (i.e. tables starting from ACT_HI_), but not from runtime tables (i.e. tables starting from ACT_RU_).
```
Comment (yanavasileva) [inline content/user-guide/process-engine/batch-operations.md:172]: `_XXX_` is the markdown for italic style, so you can use single quotes in this situation to display the text. Also, I suggest emphasizing that we talk about _database_ tables.
```suggestion
Setting removal time to running process instances would delete data from historic database tables (i.e. tables starting from `ACT_HI_*`), but not from runtime database tables (i.e. tables starting from `ACT_RU_*`).
```

**Review by yanavasileva — CHANGES_REQUESTED** [review]
@valeriaportolesicamunda I suggested small changes about the style and wording.

In case you want to check by yourself, how the end documentation will look like, you need to [install Hugo](https://github.com/camunda/camunda-docs-manual#installing-hugo) and [build the project](https://github.com/camunda/camunda-docs-manual#building-the-documentation) with it.

**Comment by yanavasileva** [comment]
Applied the requested changes, adjusted the commit message and merged with 8eec91c.

**Pull request closed without merging**